cfgov/unprocessed/apps/regulations3k/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "cfpb-regulations-3000",
+  "name": "eRegulations",
   "description": "The consumerfinance.gov/eregulations website.",
   "homepage": "https://www.consumerfinance.gov/eregulations/",
```

Comment:
Do we want to go with `/regulations/` everywhere for now?

Reply:
Yesh

Reply:
homepage too?